e2e/suites/graphs.test.ts

```diff
@@ -64,15 +74,16 @@ describe('Graphs in Page ', () => {
       await savePage();
     });
 
-    it('should display Bar chart graph in page with no more than a 3% difference', async () => {
+    it('should display Bar chart graph in page with no more than a 7% difference', async () => {
       const graphsPage = await displayGraph();
-      const chartContainer = ensure<ElementHandle>(
-        await graphsPage.$('.recharts-responsive-container')
+      const chartScreenshot = await getChartContainerScreenshot(
+        graphsPage,
+        '.recharts-responsive-container'
       );
+      const resizedChart = await resizeImage(chartScreenshot, 1000, 320);
```

Comment:
I don't think this approach will escalate. We need to find some way to avoid passing height and width every time. Maybe resize always to the same width while keeping the ration for the height?
cc. @daneryl

Reply:
Done